### DISCUSSION TITLE
Forecast Grid Expired

### DISCUSSION THREAD

**Discussion by Cloud9Developer** [weather-gov/api#322] [Q&A]
Hello. There is currently an error where I am getting the error message listed below. I'm assuming this is due to an outage(?) since the service was working fine last night (and prior as well). Not sure if this is the right forum to ask this, but any idea when it will be fixed?

**URL: https://api.weather.gov/gridpoints/LIX/27,110/forecast/hourly**
```
{
    "correlationId": "257ad490-4ff2-4492-9c78-808974251acd",
    "title": "Forecast Grid Expired",
    "type": "https://api.weather.gov/problems/ForecastGridExpired",
    "status": 503,
    "detail": "The requested forecast grid was issued 2020-07-07T12:17:50+00:00 and has expired.",
    "instance": "https://api.weather.gov/requests/257ad490-4ff2-4492-9c78-808974251acd"
}
```

**Comment by scadergit** [comment]
That is an outage as the API hasn't received updated forecast data from the office. Please [submit an outage report](https://weather-gov.github.io/api/reporting-issues).

**Comment by lucyleap** [comment]
I am also experiencing this and just submitted an outage report. @cph015, have you by chance heard any information on when they expect it to be back up?

**Comment by leetistan** [comment]
I got a reply last night after my outage report,

“This is a known issue that is being worked on. There is no estimated
time of return to service. Your patience is appreciated.”

**Comment by Cloud9Developer** [comment]
@lucyleap I haven't heard of an ETA yet. They are working on it though. And sometimes, my API call gets lucky and returns the proper data. But it is really spotty

**Comment by paulnadler** [comment]
There is an outage due to a network migration started July 7th. I'm getting this from the NCO status page here:
https://www.nco.ncep.noaa.gov/status/messages/

Perhaps this link can be added to the page for reporting outages. There is currently a link for planned outages but I think the unplanned ones would be useful too.

**Comment by scadergit** [comment]
The development team was just asked to confirm if forecast endpoints are working again, so looks to be fixed.

**Comment by gregawoods** [comment]
Hello,

Sorry if this is posted to the wrong place. Would the outage mentioned above also affect the APIs at [graphical.weather.gov](https://graphical.weather.gov/xml/)?

Specifically, an application I support calls the https://graphical.weather.gov/xml/sample_products/browser_interface/ndfdBrowserClientByDay.php endpoint. For the last few days it has been returning an error of `No data were found using the following input`.

**Comment by scadergit** [comment]
@gregawoods I heard management say "assume everything is related to this issue," so likely. If you're still experiencing issues with that endpoint (which technically is a different [development team called MDL](https://www.weather.gov/mdl/about_home)), please submit an outage report.
